test/gtest/uct/uct_test.cc

```diff
@@ -398,17 +398,47 @@ uct_test::entity::entity(const resource& resource, uct_iface_config_t *iface_con
     params->stats_root = ucs_stats_get_root();
     UCS_CPU_ZERO(&params->cpu_mask);
 
-    UCS_TEST_CREATE_HANDLE(uct_worker_h, m_worker, uct_worker_destroy,
-                           uct_worker_create, &m_async.m_async, UCS_THREAD_MODE_SINGLE);
+    status = UCS_TEST_CREATE_HANDLE(uct_worker_h, m_worker, uct_worker_destroy,
+                                    uct_worker_create, &m_async.m_async,
+                                    UCS_THREAD_MODE_SINGLE);
+    ASSERT_UCS_OK(status);
 
-    UCS_TEST_CREATE_HANDLE(uct_md_h, m_md, uct_md_close,
-                           uct_md_open, resource.md_name.c_str(), md_config);
+    status = UCS_TEST_CREATE_HANDLE(uct_md_h, m_md, uct_md_close,
+                                    uct_md_open, resource.md_name.c_str(),
+                                    md_config);
+    ASSERT_UCS_OK(status);
 
     status = uct_md_query(m_md, &m_md_attr);
     ASSERT_UCS_OK(status);
 
-    UCS_TEST_CREATE_HANDLE(uct_iface_h, m_iface, uct_iface_close,
-                           uct_iface_open, m_md, m_worker, params, iface_config);
+    for (;;) {
+        {
+            scoped_log_handler slh(wrap_errors_logger);
+            status = UCS_TEST_CREATE_HANDLE(uct_iface_h, m_iface, uct_iface_close,
+                                            uct_iface_open, m_md, m_worker, params,
+                                            iface_config);
+            if (status == UCS_OK) {
+                break;
+            }
+        }
+        EXPECT_EQ(UCS_ERR_BUSY, status);
+        if (params->open_mode != UCT_IFACE_OPEN_MODE_SOCKADDR_SERVER) {
+            break;
+        }
+
+        const struct sockaddr* c_ifa_addr =
+            params->mode.sockaddr.listen_sockaddr.addr;
+        struct sockaddr* ifa_addr = const_cast<struct sockaddr*>(c_ifa_addr);
+        if (ifa_addr->sa_family == AF_INET) {
```

Comment:
can do:
```
ASSERT_UCS_OK(ucs_sockaddr_set_port(ifa_addr, ntohs(ucs::get_port())));
```

but if we will use `ucs_sockaddr_get_port` inside `ucs::get_port()`, we may avoid calling `ntohs(...)` here

Reply:
hmm... why ucs_sockaddr_get_port defines port as unsigned? in_addr_t is uint16_t

Reply:
> why ucs_sockaddr_get_port defines port as unsigned?

yes, then it's better to have `uint16_t`. thanks for catching this!